cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1507,11 +1511,39 @@ protected String getServiceAccountName() {
         return varList;
     }
 
+    /**
+     * Validates provided OAuth configuration. Throws InvalidResourceException when OAuth configuration contains forbidden combinations.
+     *
+     * @param oAuth     OAuth type authentication object
+     */
+    private static void validateOauth(KafkaListenerAuthenticationOAuth oAuth) {
```

Comment:
I think this needs an additional check that one of Introdpection URI or JWKS URI are set.